pkg/resourcemanager/manager/managedresources.go

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Can we merge/consolidate this package with `pkg/utils/managedresources`?

Reply:
I have added it to my list of follow-ups. There are quite some more things that can be consolidated now that g/grm is part of g/g, especially w.r.t. the `extensions` directory as well.